requirements.txt

```
@@ -1,7 +1,6 @@
 cycler
-event-model>=1.5.0,!=1.8.0
+event-model==1.10.0rc1
```

```suggestion
event-model>=1.10.0rc1
```

Use a lower bound so that when we do an event-model final we don't have to re-pin this!